libs/perun/login/src/lib/login-screen-base/login-screen-base.component.ts

```diff
@@ -25,11 +25,11 @@ export class LoginScreenBaseComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
-    this.headerBackgroundColor = this.storeService.get('theme', 'nav_bg_color') as string;
-    this.headerTextColor = this.storeService.get('theme', 'nav_text_color') as string;
-    this.logo = this.sanitizer.bypassSecurityTrustHtml(this.storeService.get('logo') as string);
+    this.headerBackgroundColor = this.storeService.getProperty('theme').nav_bg_color;
+    this.headerTextColor = this.storeService.getProperty('theme').nav_text_color;
+    this.logo = this.sanitizer.bypassSecurityTrustHtml(this.storeService.getProperty('logo'));
     this.textColor = this.headerTitle
-      ? (this.storeService.get('theme', 'header_text_color') as string)
+      ? this.storeService.getProperty('theme').header_text_color
```

Comment:
You can delete `header_text_color` from `ConfigProperties.ts`.
```suggestion
      ? this.storeService.getProperty('theme').nav_text_color
```